pkg/agent/cniserver/interface_configuration_linux.go

```diff
@@ -281,16 +337,21 @@ func (ic *ifConfigurator) configureContainerLink(
 	containerNetNS string,
 	containerIfaceName string,
 	mtu int,
-	sriovVFDeviceID string,
+	brSriovVFDeviceID string,
+	podSriovVFDeviceID string,
 	result *current.Result,
 ) error {
-	if sriovVFDeviceID != "" {
+	if brSriovVFDeviceID != "" {
 		if !ic.isOvsHardwareOffloadEnabled {
 			return fmt.Errorf("OVS is configured with hardware offload disabled, but SR-IOV VF was requested; please set hardware offload to true via antrea yaml")
 		}
-		klog.V(2).Infof("Moving SR-IOV %s device to network namespace of container %s", sriovVFDeviceID, containerID)
+		klog.V(2).Infof("Moving SR-IOV %s device to network namespace of container %s", brSriovVFDeviceID, containerID)
 		// Move SR-IOV VF to network namespace
-		return ic.configureContainerLinkSriov(podName, podNamespace, containerID, containerNetNS, containerIfaceName, mtu, sriovVFDeviceID, result)
+		return ic.configureContainerSriovLinkOnBridge(podName, podNamespace, containerID, containerNetNS, containerIfaceName, mtu, brSriovVFDeviceID, result)
+	} else if podSriovVFDeviceID != "" {
+		klog.V(2).Infof("For pod link SR-IOV interface not advertising IP address to OVS.")
```

Comment:
Nits: pod -> Pod; remove "." after "OVS" (we do not add "." in log messages).

Reply:
But does this one need to be a log, or just a comment is good ?

Reply:
Made it a comment